src/org/opendatakit/briefcase/model/RemoteFormDefinition.java

```diff
@@ -26,7 +28,7 @@
 
   public RemoteFormDefinition(String formName, String formId,
       String versionString, String downloadUrl, String manifestUrl) {
-    this.formName = formName;
+    this.formName = formName == null ? null : formName.replaceAll(BasicConsts.INVALID_CHARS, BasicConsts.EMPTY_STRING);
```

Comment:
I'm in between for this one. We could change the getFormName() implementation.

This one is used to display remote form in the pull transfer panel and won't affect the creation of the folder name. I think it would be nice to make sure it looks consistent to the user.

Reply:
I'm a no on this one. I think we should only strip things when they are needed. The form title will have a period everywhere else (e.g., Collect, Aggregate) so you aren't really making things that consistent.

Reply:
Reverting.

Reply:
Will revert once @dcbriccetti is cool with the changes.

Reply:
How about reverting now so I can see exactly what you are proposing to do?

Reply:
I definitely agree that the name should be modified only when needed, i.e., when mapping it to a file path.